compose.yml

```
@@ -4,20 +4,47 @@ services:
     tty: true
     stdin_open: true
+  db_queue:
+    image: mysql:8.0.37
+    platform: linux/amd64
+    hostname: mysql
+    volumes:
+      - db-queue-data:/var/lib/mysql　
+    environment:
+      MYSQL_ROOT_PASSWORD: ''
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+    ports: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)